src/rp/util/logic.clj

```diff
@@ -0,0 +1,7 @@
+(ns rp.util.logic)
+
+(defn xor
+  [p q]
```

Comment:
I think I see what you mean by it being a naming issue. Our use case was to validate that the client had not made an ambiguous request of `/points` with both a `bounding-box` and a `neighborhood-id`. The request must provide one or the other, but never both or neither. It seemed to be a good case for `xor`. Granted if there was a third kind of request that could be made to points, `xor` would no longer model this well.